docs/src/routes/icon-button/index.js

```diff
@@ -18,38 +20,21 @@ export default class IconButtonPage extends Component {
     this.propsTable = [
       {
         component: 'IconButton',
+        props: []
+      },
+      {
+        component: 'Button',
```

Comment:
Are you sure that this is right? I thought that has to be IconButton and IconButton.Icon

Reply:
its kinda upto us to name it

Reply:
Well naming an icon button isn't quite intuitive it's the same one as in my last comment

Reply:
https://github.com/prateekbh/preact-material-components/blob/96aba2428a7ee18e3dd474770341c0b71f214171/ts/IconButton/index.tsx#L32

It's actually named half Icon, half button in code

Reply:
This is still to change if you don't want to sell the users an icon as a button